common/lib/common-utils/src/hashFileBrowser.ts

```diff
@@ -14,6 +14,13 @@ import { IsoBuffer } from "./bufferBrowser";
  * @returns The hash of the content of the buffer
  */
 export async function hashFile(file: IsoBuffer): Promise<string> {
+    // Handle insecure contexts (e.g. running with local services)
+    // by deferring to Node version, which uses a hash polyfill
+    const subtle = crypto.subtle;
+    if (subtle === undefined) {
+        return import("./hashFileNode").then(async (m) => m.hashFile(file));
```

Comment:
how much code are we adding to a bundle?

Reply:
locally it was showing ~1.2K but it should be getting pulled in dynamically... still need to verify this part

Reply:
Is it possible for the consumer to pass this in instead? Would prefer not to have the framework bake this in